### ISSUE TITLE
Suggestion: package renaming

### ISSUE THREAD

**Issue by psaussure** [flauc/angular2-notifications#202]
Hi there. 

Since package supports Angular 2 and up, could we think about a package renaming. For example ngx-notifications?

**Comment by blackholegalaxy** [comment]
I agree with that. Along with ng2-confirmation. Could be renamed `ngx`- or `angular-`

**Comment by flauc** [comment]
Yes, I agree, I think version 1.0.0 would be a good time to do it. I will try to find the time for this next weekend.
